src/NuGet.Clients/NuGet.Console/GlobalSuppressions.cs

```diff
@@ -13,8 +13,7 @@
 [assembly: SuppressMessage("Microsoft.Design", "CA1020:AvoidNamespacesWithFewTypes", Scope = "namespace", Target = "NuGetConsole.Host.PowerShell")]
 [assembly: SuppressMessage("Usage", "VSTHRD110:Observe result of async calls", Justification = "https://github.com/NuGet/Home/issues/7674", Scope = "member", Target = "~M:NuGetConsole.Implementation.Console.ConsoleDispatcher.Start")]
 [assembly: SuppressMessage("Usage", "VSTHRD002:Avoid problematic synchronous waits", Justification = "Dispose method", Scope = "member", Target = "~M:NuGetConsole.ChannelOutputConsole.Dispose(System.Boolean)")]
-[assembly: SuppressMessage("Build", "CA2211:Non-constant fields should not be visible", Justification = "<Pending>", Scope = "member", Target = "~F:NuGetConsole.GuidList.guidNuGetOutputWindowPaneGuid")]
-[assembly: SuppressMessage("Build", "CA2211:Non-constant fields should not be visible", Justification = "<Pending>", Scope = "member", Target = "~F:NuGetConsole.GuidList.guidVsWindowKindOutput")]
+[assembly: SuppressMessage("Build", "CA2211:Non-constant fields should not be visible", Justification = "Cannot change access modifier of 'guidNuGetOutputWindowPaneGuid' because it is referred in another assembly. This variable is passed as ref paramter to VS API and hence it can't be const/readonly.", Scope = "member", Target = "~F:NuGetConsole.GuidList.guidNuGetOutputWindowPaneGuid")]
```

Comment:
This project isn't part of the NuGet SDK. Changing the field to a property should make the analyzer happy as well.

Reply:
When I changed `field` to `property` -> `public static Guid guidNuGetOutputWindowPaneGuid => Guid.Parse("CEC55EC8-CC51-40E7-9243-57B87A6F6BEB"); ` compiler raises an error in another assembly.

![image](https://user-images.githubusercontent.com/52756182/117084977-708e2800-acfd-11eb-93c2-479b8ef70ee2.png)

Reply:
I have updated `justification` in the latest commit to be clearer.

Reply:
You can use a temporary variable, so the code doesn't need to try to use ref on a property.
```cs
Guid tempGuid = Guid.Zero;
if (_vsOutputWindow.Value.GetPane(ref tempGuid, out pane) == VSConstances.S_OK)
{
  NuGetConsole.Guidlist.guidNuGetOuputWindowPaneGuid = tempGuid;
  pane.Activate();
}
```

Although the name should probably be better than `tempGuid`.

Reply:
> When I changed field to property -> public static Guid guidNuGetOutputWindowPaneGuid => Guid.Parse("CEC55EC8-CC51-40E7-9243-57B87A6F6BEB"); compiler raises an error in another assembly.

My above comment is incorrect. `public static guidNuGetOutputWindowPaneGuid` property should have both `getter` and `setter` instead of `getter` alone because of usage in another assembly. I am thinking of what would be the benefit of replacing `public static` `field` with a `property` other than making the analyzer happy. There are three instances in the product code where we have similar code that passes `guidNuGetOutputWindowPaneGuid` as `ref` parameter. I could be wrong, but I lean towards keeping the suppression with justification as is. Happy to learn more if my analysis is incorrect.

![image](https://user-images.githubusercontent.com/52756182/117187436-c233c280-ad90-11eb-9c81-d2714fca042e.png)

Reply:
>  I am thinking of what would be the benefit of replacing public static field with a property other than making the analyzer happy

This is the general convention of .NET libraries. Some people believe that fields should never, ever be public, only used privately in classes. Only make methods, properties and const available in public APIs.

The reason is that fields do not provide any way to do input validation on setting. Whereas with a property setter you can. Having a auto property that you later change to a manual property with validation is not an API breaking change, whereas changing a field to a property is.

Reply:
I believe this .NET convention of avoiding public fields is a carry over from Java Enterprise development, where properties don't exist, and public fields are strongly discouraged in favor or methods starting with `get` and `set`, since they can do the appropriate input validation.

Reply:
Thanks for the feedback Andy. Since there is no validation during `setter` I was unsure of the benefit. But I liked your suggestion because changing field to property is a breaking change, and we can add validation logic to `setter` in the future without a breaking change.

Addressed this in the latest commit. Please review.